tt_eager/tt_dnn/kernels/dataflow/moreh_common.hpp

```diff
@@ -808,6 +808,8 @@ FORCE_INLINE void generate_mask_tiles(
 }
 
 uint32_t get_tilized_idx(uint32_t h, uint32_t w) {
+    h = h % TILE_HEIGHT;
+    w = w % TILE_WIDTH;
```

Comment:
Is this function frequently called?
Then, why don't we use bit-shifting rather than expensive modulo operation?

Reply:
Since `TILE_HEIGHT` and `TILE_WIDTH` are constexpr and all operands are unsigned these modulo operations are optimized to shift operations by the compiler so I think there's no need to fix it.